data/political/2012-2021-house-senate-congressional-districts/index.html

```diff
@@ -52,18 +51,17 @@
   <figcaption class="caption__text">2012 Political Districts</figcaption>
 </figure>
 {% include abstract.html
-  name="2012 Political Districts"
-  download="https://drive.google.com/drive/folders/0ByStJjVZ7c7mMW83YklfbkxfazQ"
+  name="Current Political Districts"
```

Comment:
**suggestion**: I would feel better if we didn't imply current in the title as that could be interpreted different ways. 
```suggestion
  name="Political Districts"
```